tao_compiler/mlir/disc/transforms/tests/cpu-only-lmhlo-fusion.mlir

```diff
@@ -1,5 +1,5 @@
 // RUN: disc-opt -pass-pipeline='func.func(disc-fusion{gpu-enabled=false fusion-strategy=base})' -split-input-file %s -o - | FileCheck %s --check-prefix=BASE
-// RUN: DISC_ENABLE_COMPUTE_INTENSIVE_FUSE=1 disc-opt -pass-pipeline='func.func(disc-fusion{gpu-enabled=false fusion-strategy=stitch})' -split-input-file %s -o - | FileCheck %s --check-prefix=TRANSFORM
+// RUN: DISC_ENABLE_TRANSFORM_SCHEDULE=1 disc-opt -pass-pipeline='func.func(disc-fusion{gpu-enabled=false fusion-strategy=stitch})' -split-input-file %s -o - | FileCheck %s --check-prefix=TRANSFORM
```

Comment:
how could I distinguish one env var from the other?

Reply:
generally speaking, transform schedule could be applied to memory-intensive patterns as well.